lib/game.rb

```diff
@@ -0,0 +1,108 @@
+class Game
+
+	attr_accessor :board, :player_1, :player_2
+
+	
+	 WIN_COMBINATIONS = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]]
+	 
+	 def initialize(p1 = Player::Human.new("X"), p2 = Player::Human.new("O"), board = Board.new)
+	 	@board = board
+	 	@player_1 = p1
+	 	@player_2 = p2
+	 end
+
+	 def turn_count
+	 	board.cells.count(player_1.token) + board.cells.count(player_2.token)
+	 end
+
+	 def current_player
+	 	if turn_count == 0 || turn_count.even?
```

Comment:
Zero is an even number (or at least treated as such in Ruby) and doesn't need to be checked for separately.